.github/workflows/build_and_test.yml

```diff
@@ -357,8 +360,6 @@ jobs:
     - name: Run documentation build
       run: |
         cd docs
-        export LC_ALL=C.UTF-8
-        export LANG=C.UTF-8
```

Comment:
interesting, so export doesn't work?

Reply:
Oh it works. Problem was that we didn't set both when we run lint-python above that trigfers sphinx build ..

Reply:
I guess  we need this still.